### ISSUE TITLE
Query about transitive closure for over regulates for MF by default

### ISSUE THREAD

**Issue by ValWood** [geneontology/amigo#210]
De we want to include regulates_closure for molecular function?

This give weird results. For instance if I search for all of the gene products with
"protein kinase activity" for pombe, the list includes
protein kinase activators and inhibitors like cig2 and rum1.

I think most people would find this misleading.

Val

**Comment by kltm** [comment]
This is most likely mostly a question for @cmungall (and @hdietze), as the best place to make any change along these lines would probably be during loading.

**Comment by cmungall** [comment]
The general framework here would be to have the default closure field depend on the focus class. This will be required in any case for #201 

E.g. 

 * `c` in MF
    * Question: *which gene products enable `c`?*
    * `isa_partof_closure`
    * Why: `enables o SubClassOf -> enables`
 * `c` in BP
    * Question: *which gene products are involved in `c`?*
    * `regulates_closure`
    * Why: `enables o (part_of|regulates) -> involved_in`
 * `c` in CHEBI
    * Question: *which gene products are involved in a process that have `c` as participant?*
    * `involved_in_o_has_participant_closure` (see #201)
    * Why: `enables o (part_of|regulates) o has_participant -> involved_in_o_has_participant`

etc

Of course, the boundary between MF and BP can be arbitrary. Question for @ValWood -- what would you want the default behavior for `protein phosphorylation` to be? On the one hand, it's the exact same thing as with `protein kinase acticity`. However, there is an implicit difference in question, and I think GOC members would want kinase gene sets to exclude regulatory and phosphorylation gene sets to include regulatory. Is this just a GOC expectation? What would a wider cross-section of biologists think? Something for @dosumis to think about in the MF refactoring. Whichever way we model things, AmiGO provides the framework to answer the questions in a sound fashion.

**Comment by kltm** [comment]
#267 seems to be a reformulation.

**Comment by cmungall** [comment]
@kltm correct, looks like I repeated myself: https://github.com/geneontology/amigo/issues/267#issuecomment-152879983

My fault, should not be using the tracker as a substitute for proper requirements documentation.

TL;DR: choose the default closure type based on the type of term. We should also do more to makethe reasoning more transparent and intuitive.
